### PR TITLE
Add boxes 1.2

### PR DESCRIPTION
**Pull request by patrick330602**
This is a command line filter program that draws ASCII art boxes around your input text.

### CODE REVIEW

File: bucket/boxes.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "description": "A command line filter program that draws ASCII art boxes around your input text",
+    "version": "1.2",
+    "license": "GPLv2",
+    "url": "https://github.com/ascii-boxes/boxes/releases/download/v1.2/boxes-1.2-intel-win32.zip",
+    "homepage": "https://github.com/ascii-boxes/boxes/",
+    "hash": "6f2081a51e541920debeb779df7ea8fe91d73e3ef8bdaa75b3984b531d0271ed",
+    "extract_dir": "boxes-1.2",
+    "bin": "boxes.exe",
+    "checkver": {
```
Comment (Ash258) [inline bucket/boxes.json]: Could be simplified to `"checkver": "github"`

**Comment by patrick330602** [comment]
Changes made, would this be fine?

**Comment by Ash258** [comment]
I think [this](http://boxes.thomasjensen.com/) as homepage would be better.

File: bucket/boxes.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "description": "A command line filter program that draws ASCII art boxes around your input text",
+    "version": "1.2",
+    "license": "GPLv2",
```
Comment (r15ch13) [inline bucket/boxes.json]: Should be `GPL-2.0-or-later` (See license identifiers on https://spdx.org/licenses/)

**Pull request merged**